README.md

```diff
@@ -1,34 +1,40 @@
-# test
-
-## Project setup
-```
-yarn install
-```
-
-### Compiles and hot-reloads for development
-```
-yarn serve
-```
-
-### Compiles and minifies for production
-```
-yarn build
-```
-
-### Run your unit tests
-```
-yarn test:unit
-```
-
-### Run your end-to-end tests
-```
-yarn test:e2e
-```
-
-### Lints and fixes files
-```
-yarn lint
-```
-
-### Customize configuration
-See [Configuration Reference](https://cli.vuejs.org/config/).
+# Homework for Vue3
+
+### homework for Composition API Introduction
+1.Lets build sample app in sample-vue-app folder, follow steps in README file. After build successfully, your browser will be:
+
+You can check my folder.
+
+
+
+### homework for API-Store-Component
+1.Where is API module, Store module and Component module?
+
+The Store module is in src/stores and the Component module is in src components.
+But we don't have API module in this folder.
+
+
+2.How does the Loading work? (When you click on button - or +, loading will be displayed).
+
+When you click the button in the components of "InjectedDecrementButton.vue" and "InjectedIncrementButton.vue",
+the function of "asyncIncrement" and "asyncDecrement" starts to work.
+These functions let the "loading status" be "true" and "false".
+Btw, these functions are in the stores of "Counter.ts".
```

Comment:
loading status is set true or false, so how these values effect to the loading on view?